decode/clarity.ts

```diff
@@ -45,6 +45,10 @@ export function decode(input: string): DecodedPayload {
                 if (payload.tag === undefined) { payload.tag = []; }
                 payload.tag.push(data.decode(entry) as TagEvent);
                 break;
+            case Event.Target:
+                if (payload.target === undefined) { payload.target = []; }
+                payload.target.push(data.decode(entry) as TargetEvent);
```

Comment:
>data [](start = 36, length = 4)

Why remove this case?

Reply:
Not removing it. Moving it up and handling it differently now that it's part of data.